### PR TITLE
add(app): appflowy

### PR DESCRIPTION
**Pull request by Wraithers**
AppFlowy is an AI-powered secure workspace where you achieve more without losing control of your data

## Summary by CodeRabbit

- **New Features**
	- Updated `AppFlowy` configuration settings including application details and supported architectures.
	- Enhanced Docker setup for `AppFlowy` with updated version and configurations for network settings and service management.
	- Added a comprehensive description document for `AppFlowy`, detailing features, roadmap, community, and contribution guidelines.

### CODE REVIEW

File: apps/appflowy/config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "$schema": "../schema.json",
+  "name": "AppFlowy",
+  "available": true,
+  "exposable": true,
+  "port": 8042,
+  "id": "appflowy",
+  "tipi_version": 1,
+  "version": "0.5.7",
+  "categories": [
+    "utilities"
```
Comment (JigSawFr) [inline apps/appflowy/config.json:11]: category "ai" should also be added ;)

File: apps/appflowy/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+  appflowy:
+    container_name: appflowy
+    image: appflowyio/appflowy_client:v0.5.7
+    ports:
+      - ${APP_PORT}:80
+      - 8043:443
```
Comment (JigSawFr) [inline apps/appflowy/docker-compose.yml:8]: should be avoided or used in user-config due to potential conflicts

File: apps/appflowy/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      - ${APP_DATA_DIR}/data:/home/appflowy
+      - $HOME/.Xauthority:/root/.Xauthority:rw
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - /dev/dri:/dev/dri
+      - /var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket
+    environment:
+      - DISPLAY=${DISPLAY}
+    restart: unless-stopped
+    networks:
+      - tipi_main_network
```
Comment (JigSawFr) [inline apps/appflowy/docker-compose.yml:19]: traefik labels missing

**Review by JigSawFr — CHANGES_REQUESTED** [review]

**Comment by steveiliop56** [comment]
Abandoned

**Pull request closed without merging**